packages/edit-site/src/components/post-list/style.scss

```diff
@@ -64,6 +70,25 @@
 	overflow: hidden;
 }
 
+.dataviews-view-grid__primary-field.dataviews-view-grid__primary-field--clickable,
```

Comment:
Can this style rule be moved to dataviews instead? This would be part of how a "clickable" field works as per the layout definition of a "clickable field". Consumers shouldn't have to deal with this.

Reply:
This CSS is specifically for the post title field. The only part that might make sense to move is the `cursor: pointer`, but I suggest keeping it here for now.

Reply:
> Shouldn't we formalise how "clickable" fields look like? Making the cursor a pointer, like any clickable element, seems something we should handle for the consumers.

I'm happy to move the cursor rule.

> If the other styles are only relevant for the title field, can we scope them using the .edit-site-post-list__title? We should aim for consumer code not to have any .dataviews-* CSS.

> If the other styles are only relevant for the title field, can we scope them using the .edit-site-post-list__title? 

Overall, I agree with this. The main issue is that this means that we should add `--clickable` class directly to the field.